frontends/web/src/components/forms/radio.module.css

```diff
@@ -50,7 +50,7 @@
 /* checked */
```

Comment:
there is another one a few lines before this line (sorry can't commend there so I write here),

```
.radio input + label::before {
    border: 1px solid rgb(220, 220, 220);
```

Reply:
Same as comment above (I think the current radio button colors also work for dark mode). So we can keep it how it is now and make the color variables in a future pr.